docs/reference/release-notes/7.0.0-rc1.asciidoc

```diff
@@ -1,6 +1,203 @@
+* Suppress error message when `/proc/sys/vm/max_map_count` is not exists. {pull}35933[#35933]
+
+Infra/REST API::
+* Fix #38623 remove xpack namespace REST API {pull}38625[#38625]
+* Remove the "xpack" namespace from the REST API {pull}38623[#38623]
+
+Machine Learning::
+* [ML] Fix race condition when creating multiple jobs {pull}40049[#40049] (issue: {issue}38785[#38785])
+* ML allow aliased .ml-anomalies* index on PUT Job {pull}38821[#38821] (issue: {issue}38773[#38773])
+
+Recovery::
+* Create retention leases file during recovery {pull}39359[#39359] (issue: {issue}37165[#37165])
+
+SQL::
+* SQL: Add missing handling of IP field in JDBC {pull}40384[#40384] (issue: {issue}40358[#40358])
+* SQL: Fix metric aggs on date/time to not return double {pull}40377[#40377] (issues: {issue}39492[#39492], {issue}40376[#40376])
+* SQL: CAST supports both SQL and ES types {pull}40365[#40365] (issue: {issue}40282[#40282])
+* SQL: Fix RLIKE bug and improve testing for RLIKE statement {pull}40354[#40354] (issues: {issue}34609[#34609], {issue}39931[#39931])
+* SQL: unwrap the first value in an array in case of array leniency {pull}40318[#40318] (issue: {issue}40296[#40296])
+* SQL: Preserve original source for cast/convert function {pull}40271[#40271] (issue: {issue}40239[#40239])
+* SQL: fix LIKE function equality by considering its pattern as well {pull}40260[#40260] (issue: {issue}39931[#39931])
+* SQL: Fix issue with optimization on queries with ORDER BY/LIMIT {pull}40256[#40256] (issue: {issue}40211[#40211])
+* SQL: rewrite ROUND and TRUNCATE functions with a different optional parameter handling method {pull}40242[#40242] (issue: {issue}40001[#40001])
+* SQL: Fix issue with getting DATE type in JDBC {pull}40207[#40207]
+* SQL: Fix issue with date columns returned always in UTC {pull}40163[#40163] (issue: {issue}40152[#40152])
+* SQL: Add multi_value_field_leniency inside FieldHitExtractor {pull}40113[#40113] (issue: {issue}39700[#39700])
+* SQL: fix incorrect ordering of groupings (GROUP BY) based on orderings (ORDER BY) {pull}40087[#40087] (issue: {issue}39956[#39956])
+* SQL: Fix bug with JDBC timezone setting and DATE type {pull}39978[#39978] (issue: {issue}39915[#39915])
+* SQL: Use underlying exact field for LIKE/RLIKE {pull}39443[#39443] (issue: {issue}39442[#39442])
+
+Search::
+* Serialize top-level pipeline aggs as part of InternalAggregations {pull}40177[#40177] (issues: {issue}40059[#40059], {issue}40101[#40101])
+* CCS: skip empty search hits when minimizing round-trips {pull}40098[#40098] (issues: {issue}32125[#32125], {issue}40067[#40067])
+* CCS: Disable minimizing round-trips when dfs is requested {pull}40044[#40044] (issue: {issue}32125[#32125])
+
+
+
+[[upgrade-7.0.0-rc1]]
+[float]
+=== Upgrades
+
+Discovery-Plugins::
+* Bump jackson-databind version for AWS SDK {pull}39183[#39183]
+
+Engine::
+* Upgrade to lucene 8.0.0-snapshot-ff9509a8df {pull}39445[#39445] (issue: {issue}39350[#39350])
```

Comment:
this one is a backport and shouldn't be here, I updated the labels on github

Reply:
Addressed in https://github.com/elastic/elasticsearch/pull/40479/commits/9c0a8979144c9e291dce9ff283d9b54cc007e9ed